Makefile

```diff
@@ -1,4 +1,4 @@
-RETRY ?= 0
+RETRY ?= 3
```

Comment:
Please don't. We only applied RETRY for the cases where we know about instabilities. In all other cases we don't retry so that we would fail on the very first error.